src/os.ts

```diff
@@ -1,29 +1,32 @@
 enum OS {
   WIN32 = 'win32',
   WIN64 = 'win64',
-  MAC32 = 'mac32',
   MAC64 = 'mac64',
+  MAC_ARM64 = 'mac64',
   LINUX = 'linux',
+  UNSUPPORTED = 'unsupported',
 }
 
 const getOS = (): OS => {
-  const platform: string = process.platform;
-  const arch: string = process.arch;
+  const platform = process.platform;
+  const arch = process.arch;
```

Comment:
The string type was removed so that TypeScript can infer the more accurate and narrow type from `process`